src/main/java/com/hubspot/jinjava/interpret/UnknownTokenException.java

```diff
@@ -6,7 +6,7 @@
   private final String token;
 
   public UnknownTokenException(String token, int lineNumber, int startPosition) {
-    super("Unknown token found: " + token, lineNumber, startPosition);
+    super("Unknown token found: " + token.trim(), lineNumber, startPosition);
```

Comment:
do we still need the `trim()`

Reply:
yes. since there was a test that ended up failing because of incorrect padding.
the only way was to trim the token to get rid of the extra space and make the tests pass

Reply:
[this test](https://github.com/HubSpot/jinjava/pull/345/files#diff-b386f56a1fcb39f707f026174037eab7R152) was failing since the value was coming out to be
`"Unknown token found:  bad " `, where as the expected value is `"Unknown token found: bad"`

therefore I trimmed the token

Reply:
You can use `hasMessageContaining` instead.